csunplugged/tests/resources/views/test_binary_to_alphabet.py

```diff
@@ -0,0 +1,154 @@
+from django.urls import reverse
+from tests.BaseTestWithDB import BaseTestWithDB
+from tests.resources.ResourcesTestDataGenerator import ResourcesTestDataGenerator
+from utils.create_query_string import query_string
+
+
+class BinaryToAlphabetResourceViewTest(BaseTestWithDB):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.test_data = ResourcesTestDataGenerator()
+        self.language = "en"
+
+    def test_binary_to_alphabet_resource_form_view(self):
+        resource = self.test_data.create_test_resource(
+            "binary-to-alphabet",
+            "Binary To Alphabet",
+            "resources/binary-to-alphabet.html",
+            "binary_to_alphabet.py"
+        )
+        kwargs = {
+            "resource_slug": resource.slug
+        }
+        url = reverse("resources:resource", kwargs=kwargs)
+        response = self.client.get(url)
+        self.assertEqual(200, response.status_code)
+
+    def test_binary_to_alphabet_resource_generation_student_a4(self):
```

Comment:
For consistency, it might be good to test valid generation using the test that creates combinations from the `valid_options` method. The `header_text` values should also be added to the valid options (could make a utility function for getting valid options including header text option).